### ISSUE TITLE
Soulcalibur renders at lower resolution than 480p when Full Framebuffer is enabled.

### ISSUE THREAD

**Issue by levonnn** [flyinghead/flycast#1775]
**Please Note: This form is the minimum required information for submitting bugs.**  
**Removing this form may lead to your issue being closed until it is completed.**

Platform / OS / Hardware: 
Win 10 64

Flycast version: 
1.17.0 (15617c1)

Hardware: 
RX580, i7 920, 16 Gb RAM

**_Description of the Issue_**
All setting on default. If Full fremebuffer emulation is enabled, the entire game is rendered in, what feels like, 240p instead of 480p.

**_Screenshots_**
![Soulcalibur (NTSC-J)-241211-174831](https://github.com/user-attachments/assets/eeaed6b2-9f92-415b-8474-1f091cea92f5)

**Comment by flyinghead** [comment]
This is correct: in 480i mode, the game uses a 1/2 vertical scale factor to convert from 480p to 240p.
This doesn't happen in VGA mode.
